src/apps/project-home/ProjectHome.tsx

```diff
@@ -71,7 +69,7 @@ export const ProjectHome = (props: ProjectHomeProps) => {
 
   const { addDocumentIds } = useDocumentList(
     project.id,
-    defaultContext!.id,
+    defaultContext?.id,
```

Comment:
Just for my understanding: can the defaultContext ever be undefined? (Or would this point to an integrity problem in the DB?)

Reply:
Yes. If the user is not a member of a layer in the default context, then it is null as it does not pass RLS.  It hasn't been a problem I guess since that user should. not see any documents anyway, but if use a student user in Production you will see an error thrown every time they go into the project home.